cpp/src/arrow/flight/sql/odbc/tests/odbc_test_suite.cc

```diff
@@ -169,6 +173,13 @@ void FlightSQLOdbcV2RemoteTestBase::SetUp() {
   connected_ = true;
 }
 
+void FlightSQLOdbcHandleRemoteTestBase::SetUp() {
+  ODBCRemoteTestBase::SetUp();
```

Comment:
You should add the following after SetUp:

if (skipping_test_) {
    return;
  }